File: build_management/bazel_primer/python_tutorial/stage2/main/hello_world.py
from datetime import datetime
from hello_greet import get_greet

def print_localtime():
    now = datetime.now()
    print(now.strftime("%a %b %d %H:%M:%S %Y"))	

if __name__ == "__main__":
    who = "world"
    print(get_greet(who))
    print_localtime()

File: build_management/bazel_primer/c_tutorial/stage2/main/hello_world.c
#include "hello_greet.h"
#include <stdio.h>
#include <time.h>

void print_localtime() {
  time_t t;
  time(&t);
  printf("\n%s", ctime(&t));
}

int main(int argc, char** argv) {
  char* who = "world";
  printf("%s", get_greet(who));
  print_localtime();
  return 0;
}
